### ISSUE TITLE
Run migration was failed

### ISSUE THREAD

**Issue by nuxzero** [aschmelyun/docker-compose-laravel#19]
Hello,

This project looks pretty cool and helpful to me. 

So, I'm trying to setup the Laravel project by using your **Docker Compose**. Everything looks fine the database was created but when I run the migration command. It's occurred an error in below.

**.env** configs:

```
DB_CONNECTION=mysql
DB_HOST=127.0.0.1
DB_PORT=3306
DB_DATABASE=homestead
DB_USERNAME=homestead
DB_PASSWORD=secret
```

Then run command:

```
docker-compose run --rm artisan migrate
```

Here is the error:

```
Starting docker-compose-laravel_mysql_1 ... done

   Illuminate\Database\QueryException 

  SQLSTATE[HY000] [2002] Connection refused (SQL: select * from information_schema.tables where table_schema = homestead and table_name = migrations and table_type = 'BASE TABLE')

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:670
    666|         // If an exception occurs when attempting to run a query, we'll format the error
    667|         // message to include the bindings with SQL, which will make this exception a
    668|         // lot more helpful to the developer instead of just the database's errors.
    669|         catch (Exception $e) {
  > 670|             throw new QueryException(
    671|                 $query, $this->prepareBindings($bindings), $e
    672|             );
    673|         }
    674| 

  • Database name seems incorrect: You're using the default database name `homestead`. This database does not exist.
    
    Edit the `.env` file and use the correct database name in the `DB_DATABASE` key. 
    https://laravel.com/docs/master/database#configuration
```

Could you help to solve this issue please?

Best regards,

Natthawut

**Comment by Bert210** [comment]
Docker containers run on their own network.

If your using the default docker-compose.yml file it will be `mysql`
So in you .env file change
`DB_HOST=127.0.0.1`
to
`DB_HOST=mysql`

**Comment by nuxzero** [comment]
@Bert210 it's working 🎉 . Thanks a lot for your help.

**Comment by golchha21** [comment]
> 127

The  DB_HOST is set to mysql but I still am getting the error,

**Comment by yassine-ouaarab** [comment]
idek fiih

**Comment by dogatonic** [comment]
I had a similar problem. I could not get `docker-compose run --rm artisan migrate` to work and it kept giving me a similar error. Then I tried something else - I cannot explain why this command worked for me, but it did...
`docker-compose exec php php /var/www/html/artisan migrate`

After I ran that the first time... I could then run the migrate command using `docker-compose run --rm artisan migrate`.

If that sounds crazy, I agree, but that's what happened.
